### PR TITLE
Migrate QB summarization sidebar to MLv2

### LINKED ISSUE #31092: [MLv2] Port summarization sidebar GUI
We need to port this UI for aggregations and breakouts:

image

You get here by clicking the `Summarize` button in the upper-right when looking at query results.

We should probably sequence this after we finish the notebook editor versions of this stuff #30509 and #30511

### PR DESCRIPTION
**Pull request by kulyk**
Closes #31092

Migrates query builder's summarization sidebar to MLv2

### To Verify

1. Open a structured (non-native) question
2. Click the "Summarize" button at the top right
3. Ensure the behavior matches `master`

### CODE REVIEW

File: .vscode/settings.json
Comment (kulyk) [inline .vscode/settings.json:1]: Prettier did this, should I keep it here?

**Review by ranquild — COMMENTED** [review]
Code-wise looks good. Found several issues during testing:

1) It doesn't seem to possible to add a custom expression. Popover is displayed incorrectly, the cursor is not visible, and even with the correct expression copy-pasted it doesn't allow to add it.

Screenshot 2023-06-30 at 15 30 21
Screenshot 2023-06-30 at 15 30 38


2) Adding metrics as aggregations doesn't seem to work

Screenshot 2023-06-30 at 15 35 06

**Comment by kulyk** [comment]
@ranquild thanks, great catch! Please see [`3c72dd6`](https://github.com/metabase/metabase/pull/31564/commits/3c72dd68283b688450ac3d2fc4a0164b396ed978) — it should fix the expression issue. We actually don't allow adding inline expressions from the sidebar and I forgot to disable it for "new aggregation" component. As for the metrics, it turned out to be a BE issue, filed #32049

**Comment by kulyk** [comment]
@ranquild alright, both issues should be fixed now

**Review by ranquild — APPROVED** [review]

**Pull request merged**